src/app/components/extendedScreenContainer/index.tsx

```diff
@@ -6,11 +6,13 @@ import styled from 'styled-components';
 const ExtendedScreenRouteContainer = styled.div((props) => ({
   display: 'flex',
   flexDirection: 'column',
-  height: '100vh',
+  height: '100%',
+  minHeight: '100vh',
```

Comment:
same question as below. 

did you check that this doesn't break the other screens which open up via options.html?
e.g. any of the collectibles screens when you click 'Open gallery' button, or the ledger send flows